Cargo.toml

```diff
@@ -44,7 +44,7 @@ futures = "0.3.15"
 futures-util = "0.3.28"
 getset = "0.1.2"
 group = "0.13"
-hdwallet = "0.4.1"
+hdwallet = { git = "https://github.com/fluidvanadium/hdwallet.git", tag = "update_secp256k1_to_0.27.0" }
```

Comment:
Should this be a fork, inside zingolabs?

Reply:
doesnt matter. the important bit is that its a tag